src/client/src/ui/spotTile/components/PriceButton.tsx

```diff
@@ -19,12 +19,12 @@ export const TradeButton = styled('button')<{ direction: string }>`
   padding: 0.5rem 1.5rem;
 
   .spot-tile:hover & {
```

Comment:
We have the power! — I guess we can use component selectors next time this needs a pass, eh?